Makefile

```diff
@@ -137,14 +137,17 @@ clean.bazel: ; $(info $(H) cleaning...)
 clean.go: ; $(info $(H) cleaning...)
 	$(eval GO_CLEAN_FLAGS := -i -r)
 	$(Q) $(GO) clean $(GO_CLEAN_FLAGS)
+	$(MAKE) clean -C mixer
```

Comment:
can probably be a foreach subdirs and generalized for all/most targets at some point but later is fine